tests/scripts/all.sh

```diff
@@ -2345,10 +2351,10 @@ config_psa_crypto_hash_use_psa () {
     scripts/config.py unset MBEDTLS_ENTROPY_C
     scripts/config.py unset MBEDTLS_ENTROPY_NV_SEED # depends on ENTROPY_C
     scripts/config.py unset MBEDTLS_PLATFORM_NV_SEED_ALT # depends on former
-    # Also unset MD_C and things that depend on it;
-    # see component_test_crypto_full_no_md.
+    # Also unset MD_C and things that depend on it.
     if [ "$DRIVER_ONLY" -eq 1 ]; then
         scripts/config.py unset MBEDTLS_MD_C
+        scripts/config.py unset MBEDTLS_MD_LIGHT
```

Comment:
this will probably fail, since there's no MBEDTLS_MD_LIGHT define in the config now.

Reply:
> this will probably fail, since there's no MBEDTLS_MD_LIGHT define in the config now.

I tested it and `config.py` doesn't complain if there's a call to unset a define that doesn't exist, only set complains. Still, there's no `MBEDTLS_MD_LIGHT` in the config file anymore.

Reply:
`config.py unset FOO` doesn't require `FOO` to exist (originally for backward compatibility with `config.pl`; I don't know if our test scripts currently depend on it).

If we want to ensure that a particular build does not enable `MBEDTLS_MD_LIGHT`, we can grep for an `mbedtls_md_xxx` symbol in `libmbedcrypto.a`.

Reply:
Agreed, even if `config.py` happens not to complain, this command is confusing, I'll remove it. (I was planning to but apparently forgot at some point.)

> If we want to ensure that a particular build does not enable `MBEDTLS_MD_LIGHT`, we can grep for an `mbedtls_md_xxx` symbol in `libmbedcrypto.a`.

Yes, that's what we're doing in this component (which is temporary, we're going to have `MD_LIGHT` enabled there at some point in the future).